### DISCUSSION TITLE
Compressions has never worked for me

### DISCUSSION THREAD

**Discussion by mareqj** [iterate-ch/cyberduck#14686] [Q&A] [answered]
I am using cyberduck for years, and I accepted that its compression is non-existent because it simply doesn't work.
Since download of non-compressed folders is eeeextremly slow, I decided to ask why compression doesn't work? I have compression options on SFTP connections, but they always give connection failed error. There wasn't a single case in years, when that worked. Why?

**Comment by dkocher** [comment]
Compression using `zlib` is enabled [^1] by default.

[^1]: https://github.com/iterate-ch/cyberduck/blob/84607e0b001aedb4d6fe967a11e2f68df0606b8e/defaults/src/main/resources/default.properties#L604

**Reply by mareqj** [reply]
It's only for ssh?

**Reply by tatze96** [reply, marked as answer]
Creating and expanding ZIP and TAR Archives is limited to FTP and SFTP connections, given that the server has support for it.
